recipes/cpprestsdk/all/conanfile.py

```diff
@@ -68,6 +78,13 @@ def _configure_cmake(self):
         self._cmake.definitions["WERROR"] = False
         self._cmake.definitions["CPPREST_EXCLUDE_WEBSOCKETS"] = not self.options.with_websockets
         self._cmake.definitions["CPPREST_EXCLUDE_COMPRESSION"] = not self.options.with_compression
+        if self.options.get_safe("pplx_impl"):
+            self._cmake.definitions["CPPREST_PPLX_IMPL"] = self.options.pplx_impl
+        if self.options.get_safe("http_client_impl"):
+            self._cmake.definitions["CPPREST_HTTP_CLIENT_IMPL"] = self.options.http_client_impl
+        if self.options.get_safe("http_listener_impl"):
+            self._cmake.definitions["CPPREST_HTTP_LISTENER_IMPL"] = self.options.http_listener_impl
```

Comment:
```suggestion     
        self._cmake.definitions["CPPREST_PPLX_IMPL"] = self.options.get_safe("pplx_impl", False)
        self._cmake.definitions["CPPREST_HTTP_CLIENT_IMPL"] = self.options.get_safe("http_client_impl", False)
        self._cmake.definitions["CPPREST_HTTP_LISTENER_IMPL"] = self.options.get_safe("http_listener_impl", False)
```

Reply:
I don't think that has the same effect? I don't want to set the CMake definitions if the Conan option is not specified.

Reply:
Same effect, but simplified. Setting to False is not a problem, if you are following the correct default options.

Reply:
@uilianries 
These options are no bools